src/keybindings.h

```diff
@@ -274,6 +274,8 @@ enum GeanyKeyBindingID
 	GEANY_KEYS_FORMAT_SENDTOCMD8,				/**< Keybinding. */
 	GEANY_KEYS_FORMAT_SENDTOCMD9,				/**< Keybinding. */
 	GEANY_KEYS_EDITOR_DELETELINETOBEGINNING,	/**< Keybinding. */
+	GEANY_KEYS_DOCUMENT_STRIPTRAILINGSPACES,	/**< Keybinding. 
+												 * @since 1.34   */
```

Comment:
Actually it should be a `GEANY_API_VERSION` number

Reply:
Well, actually both, sorry: `@since 1.34 (API 23X)` is the best.  But yours was historically OK, so maybe it's fine -- and I can change this myself when merging anyway.

Reply:
So should I increase the API version also (and add it to the since-comment)?

Reply:
@LarsGit223 yes.

Though as I just updated it, maybe some people would argue that we could re-use this… which adds less noise but is less precise.  Opinions @elextr @codebrainz?

Reply:
Well, now @b4n has brought it up, yes this should increment the API, but probably not the ABI since although GEANY_KEYS_COUNT will change it says "must not be used in plugins".  And IMO the `@since` should be the API, thats what plugin writers care about and need to specify to ensure a compatible Geany version.  Theoretically the API could change several times in a release cycle, though that was more of a problem back when it was years (literally) between releases.

Reply:
I like the current practice of putting both the Geany version and the API version since as a plugin developer you're forced to know both and not have to reverse engineer one from the other is useful (which is why this was started IIRC). You need to know the Geany version to know which releases of Geany the plugin is actually compatible with, but you also need to know the API version to satisfy the `plugin_version_check()` thing in the plugin API.

In a perfect world we'd just get rid of the plugin API version and use only the release version to make the whole concept simpler. I have [an old commit](https://github.com/codebrainz/geany/commit/2a678cf0513fe7fd5163708c586adf5c545ed589) that added this in a backwards compatible way. IIRC there was an objection because of the theoretical case where some developer might want to use intermediate API versions in-between release without having to keep their feature branch up-to-date or something.

Reply:
@codebrainz right now I kind of like your proposal.  Yeah it's less specific, but avoids neverending noise of updating API version at each API-facing change.  I'd vote for that :)